### PR TITLE
Update issue-guide.md

### PR DESCRIPTION
**Pull request by ViktorHofer**
Now I finally now why I never got emails for the infra area.

### CODE REVIEW

**Review by maryamariyan — APPROVED** [review]

**Comment by maryamariyan** [comment]
I would want to be relying on this issue-guide to use the bot to set users as reviewers.
should @karelz 's name be left out as well for area-infra?

**Comment by ViktorHofer** [comment]
not sure, he's the official infra owner.

**Comment by maryamariyan** [comment]
that's alright then.

File: Documentation/project-docs/issue-guide.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Areas are tracked by labels area-&#42; (e.g. area-System.Collections). Each area
 
 | Area                                                                                          | Owners / experts | Description |
 |-----------------------------------------------------------------------------------------------|------------------|-------------|
-| [area-Infrastructure](https://github.com/dotnet/corefx/labels/area-Infrastructure)            | [@ericstj](https://github.com/ericstj), [@karelz](https://github.com/karelz), [@wtgodbe](https://github.com/wtgodbe) | Covers:<ul><li>Packaging</li><li>Build and test infra for CoreFX repo</li><li>VS integration</li></ul><br/> |
+| [area-Infrastructure](https://github.com/dotnet/corefx/labels/area-Infrastructure)            | [@ViktorHofer](https://github.com/ViktorHofer), [@ericstj](https://github.com/ericstj), [@karelz](https://github.com/karelz) | Covers:<ul><li>Packaging</li><li>Build and test infra for CoreFX repo</li><li>VS integration</li></ul><br/> |
```
Comment (stephentoub) [inline Documentation/project-docs/issue-guide.md]: And removing Will is correct?

Comment (ViktorHofer) [reply Documentation/project-docs/issue-guide.md]: yes, as he is currently fully booked with aspnetcore. cc @wtgodbe

**Pull request closed without merging**